### ISSUE TITLE
fastapi: root_path cannot contain a /

### ISSUE THREAD

**Issue by sbidoul** [OCA/rest-framework#446]
I'd like to be able to use a root path such as /myapi/v1

This does not work because the fastapi dispatcher takes the first component of the url as the root path to search the endpoint

https://github.com/OCA/rest-framework/blob/2380e306e896c732ce20c66000f5bc094731920b/fastapi/fastapi_dispatcher.py#L32

**Comment by oerp-odoo** [comment]
Yeah I was confused why my endpoint was not working as this is not documented in odoo fastapi (or maybe I did not see it?).

`/api/demo` gives internal server error, and `/api_demo` works.
